app/request_handler.py

```diff
@@ -184,31 +184,35 @@ def _guessYelpId(placeName, lat, lon):
         return cachedId
 
     opts = {
-      # 'term': placeName,
+      'term': placeName,
       'limit': 20,
-      'radius_filter': 1000,
+      'radius_filter': 100,
       #'radius_filter': 100,
       #'sort_by': 'distance',
       'sort': 1,
     }
     r = yelpClient.search_by_coordinates(lat, lon, **opts)
-    if len(r.businesses) > 0:
-        location = (lat, lon)
-        biz = min(r.businesses, key=lambda b: 
-            geo.distance(location, 
-                         (b.location.coordinate.latitude, b.location.coordinate.longitude))
-        )
-        log.debug("%s --> %s" % (placeName, biz.name))
-        researchVenue(biz)
-
-        # Add bizId to cache
-        record = { "cache/" +  safePlaceName : str(biz.id) }
-        db.child(eventsTable).update(record)
-
-        return biz.id
-    else:
-        log.info("Can't find %s" % placeName)
-        return None
+    try:
+        if len(r.businesses) > 0:
+            location = (lat, lon)
+            biz = min(r.businesses, key=lambda b:
+                geo.distance(location,
+                             (b.location.coordinate.latitude, b.location.coordinate.longitude))
+            )
+            log.debug("%s --> %s" % (placeName, biz.name))
+            researchVenue(biz)
+
+            # Add bizId to cache
+            record = { "cache/" +  safePlaceName : str(biz.id) }
+            db.child(eventsTable).update(record)
+
+            return biz.id
+        else:
+            log.info("Can't find %s" % placeName)
+            return None
+    except:
+        log.error("Error parsing yelp businesses for " + placeName)
+
```

Comment:
Will do - I had originally added this check but then decided that I just didn't trust what yelp returns, and decided to wrap everything :P But good point, my try/catch should have been inside the for loop. I'll make this change and then repush.

The reason I originally added this was that while fixing timezones, I found some differences while running the scripts/print-gcal-eventful-timezones code - adding both the term and the radius greatly greatly increased the actual correct card the event was located at, although it missed some places...but now I remember that we talked about that tradeoff yesterday and decided it was worth it to have more places rather than have them be exactly correct.

I'd like to revisit this at some point (I think we could do some fuzzy matching with scores to make this better) but okay, I think for the Hawaii prototype this is fine.